modules/drivers/snowflake/src/metabase/driver/snowflake.clj

```diff
@@ -406,28 +405,62 @@
   (sql-jdbc/query driver database {:select [:*]
                                    :from   [[(qp.store/with-metadata-provider (u/the-id database)
                                                (sql.qp/->honeysql driver table))]]}))
+(defn- show-command-sql
+  ;; IMPORTANT: prefer using this command over using the JDBC `.getTables` method because the jdbc is buggy
+  ;; it works sometimes but other times randomly.
+  ;; we have tried to use it but people reported that some schemas disappear on their instances.
+  ;; See https://discourse.metabase.com/t/snowflake-broken-in-0-48-1/63849
+  [object-type & {:keys [like database schema]}]
+  ;; in case the name is escaped for schema using escape-name-for-metadata, we undo that just to be sure
```

Comment:
Not really super important but I really don't think you need to unescape stuff since the only place that uses it is the default implementation of `describe-database`, and since you're doing a custom implementation you shouldn't need to worry about it

Reply:
Either way it's always better to not escape stuff in the first place than to escape it and unescape it, less moving parts and less ways to accidentally do something wrong